udt4/src/Makefile

```diff
@@ -18,15 +18,15 @@ ifeq ($(arch), POWERPC)
    CCFLAGS += -mcpu=powerpc
 endif
 
-ifeq ($(arch), SPARC)
+ifeq ($(arch), sparc64)
    CCFLAGS += -DSPARC
 endif
 
-ifeq ($(arch), IA64)
+ifeq ($(arch), ia64)
    CCFLAGS += -DIA64
 endif
 
-ifeq ($(arch), AMD64)
```

Comment:
is this correct across archs? is `amd64` not used in some?

maybe use something like in https://github.com/jbenet/go-env-sh/blob/master/go-platform